### ISSUE TITLE
Does anyone occur white cells by using 1.6.11?

### ISSUE THREAD

**Issue by yihuds** [MortimerGoro/MGSwipeTableCell#354]
When I updated the 1.6.4 to 1.6.11, we found the cell on the UITableView, some become white?
![image](https://user-images.githubusercontent.com/7609478/85936530-4d9d2580-b92e-11ea-89d8-a9ea91d9f9d8.png)

**Comment by serhiikrivtsov** [comment]
> When I updated the 1.6.4 to 1.6.11, we found the cell on the UITableView, some become white?
> ![image](https://user-images.githubusercontent.com/7609478/85936530-4d9d2580-b92e-11ea-89d8-a9ea91d9f9d8.png)

I'm facing exactly the same issue. It is reproducible very rarely. Looks like all subviews of contentView are hidden and not shown back again. Even after reusing of the cell subviews are not becoming visible.

@yihuds do you have any thought why it might happen?

**Comment by yihuds** [comment]
yep, I think maybe have some problem with the overlay. but I will reset the version to 1.6.4 because I don't know how to fix this bug...
@serhiikrivtsov

**Comment by yihuds** [comment]
I had compared the 1.6.4   1.6.9     1.6.11
I think this issue may be caused by fix https://github.com/MortimerGoro/MGSwipeTableCell/issues/334 on 1.6.9.

**Comment by yihuds** [comment]
@MortimerGoro Could you help to take a look? thanks

**Comment by MortimerGoro** [comment]
@yihuds can you send me a small sample to reproduce the issue? or can it be reproduced on the samples in the repo?

**Comment by yihuds** [comment]
Thanks @MortimerGoro 
It's hard to reproduce the issue, the probability is less than one in a thousand. So sorry I couldn't make a sample to reproduce...
We use it very normal...
![image](https://user-images.githubusercontent.com/7609478/90839687-b0d59200-e38a-11ea-9385-79aa05c041b7.png)
